### PR TITLE
Move workflow samples to external files

### PR DESCRIPTION
**Pull request by mshima**
Allow jhipster-bom project to auto pick samples changes.

---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [ ] Tests are added where necessary
- [ ] The JDL part is updated if necessary
- [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) is updated if necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (below reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: .github/workflows/angular.yml [hunk 1]
```diff
@@ -168,10 +94,11 @@ jobs:
...
-          jdl-entities-sample: ${{ matrix.jdl-entity }}
-          application-sample: ${{ matrix.jhi-app-sample || matrix.app-sample }}
-          application-environment: ${{ matrix.environment }}
+          application-sample: ${{ matrix.jhi-app-sample || matrix.app-sample || 'jdl' }}
+          entities-sample: ${{ matrix.entity || 'none' }}
+          jdl-entities-sample: ${{ matrix.jdl-entity || '' }}
+          jdl-sample: ${{ matrix.jdl-samples || '' }}
+          application-environment: ${{ matrix.environment || 'prod' }}
           application-packaging: ${{ (matrix.war == 1 && 'war') || 'jar' }}
           enable-testcontainers: ${{ matrix.testcontainers == 1 }}
```
Comment (mshima) [inline .github/workflows/angular.yml:103]:
```suggestion
          enable-testcontainers: ${{ matrix.testcontainers != 0 }}
```

File: .github/workflows/react.yml [hunk 2]
```diff
@@ -123,9 +94,11 @@ jobs:
...
-          entities-sample: ${{ matrix.entity }}
-          application-sample: ${{ matrix.app-sample }}
-          application-environment: ${{ matrix.environment }}
+          application-sample: ${{ matrix.jhi-app-sample || matrix.app-sample || 'jdl' }}
+          entities-sample: ${{ matrix.entity || 'none' }}
+          jdl-entities-sample: ${{ matrix.jdl-entity || '' }}
+          jdl-sample: ${{ matrix.jdl-samples || '' }}
+          application-environment: ${{ matrix.environment || 'prod' }}
           application-packaging: ${{ (matrix.war == 1 && 'war') || 'jar' }}
           enable-testcontainers: ${{ matrix.testcontainers == 1 }}
```
Comment (mshima) [inline .github/workflows/react.yml:103]:
```suggestion
          enable-testcontainers: ${{ matrix.testcontainers != 0 }}
```

File: .github/workflows/vue.yml [hunk 3]
```diff
@@ -132,9 +94,11 @@ jobs:
...
-          entities-sample: ${{ matrix.entity }}
-          application-sample: ${{ matrix.app-sample }}
-          application-environment: ${{ matrix.environment }}
+          application-sample: ${{ matrix.jhi-app-sample || matrix.app-sample || 'jdl' }}
+          entities-sample: ${{ matrix.entity || 'none' }}
+          jdl-entities-sample: ${{ matrix.jdl-entity || '' }}
+          jdl-sample: ${{ matrix.jdl-samples || '' }}
+          application-environment: ${{ matrix.environment || 'prod' }}
           application-packaging: ${{ (matrix.war == 1 && 'war') || 'jar' }}
           enable-testcontainers: ${{ matrix.testcontainers == 1 }}
```
Comment (mshima) [inline .github/workflows/vue.yml:103]:
```suggestion
          enable-testcontainers: ${{ matrix.testcontainers != 0 }}
```

**Comment by DanielFran** [comment]
@mshima Seems good. Is it missing webflux?

**Comment by mshima** [comment]
I think we should move webflux samples to angular/react/vue workflows

**Review by DanielFran — APPROVED** [review]

**Pull request merged**